modules/dynamic-provisioning-azure-file-definition.adoc

```diff
@@ -65,3 +65,4 @@ volumeBindingMode: Immediate
 `skuName` and `location` are ignored. If no storage account is provided, then
 the storage class searches for any storage account that is associated with the
 resource group for any accounts that match the defined `skuName` and `location`.
+<5> Using `WaitForFirstConsumer` is strongly recommended. This provisions the volume while allowing enough storage to schedule the pod on a free worker node from an available zone.
```

Comment:
Looks like it's unnecessary.

Reply:
IIUC, the "is recommended" note should only be used for Azure Disk. So, I have removed it here and relocated to the Azure Disk definition example